benchmark/cupy/profile_communication.py

```diff
@@ -245,6 +247,13 @@ def sync(self):
                 "FI_EFA_USE_DEVICE_RDMA": "1",
                 "LD_LIBRARY_PATH": os.environ.get("LD_LIBRARY_PATH", ""),  # For libnccl-net.so
             }
+        elif args.ib:
+            env_vars = {
+                "NCCL_SOCKET_NTHREADS": "4",
+                "NCCL_NSOCKS_PERTHREAD": "4",
+                "NCCL_IB_HCA": "ibp",
```

Comment:
this is hard code? because other machines might have different IB interface, e.g., `mlx5`

Reply:
that's right, for CoreWeave mlx5 is for ethernet and ibp is for inifiniband, but i remember for nvidia's case it could be different depending on host type